package.json

```diff
@@ -56,69 +55,83 @@
+    "@compodoc/compodoc": "^1.1.11",
     "@types/chai": "^4.1.1",
-    "@types/html-webpack-plugin": "^2.28.0",
-    "@types/mocha": "^2.2.46",
-    "@types/node": "^9.3.0",
-    "@types/sinon": "^4.1.3",
-    "@types/sinon-chai": "^2.7.26",
-    "@types/webpack": "^3.8.3",
+    "@types/html-webpack-plugin": "^3.2.3",
+    "@types/mocha": "^8.0.3",
+    "@types/node": "^14.11.5",
+    "@types/sinon": "^9.0.8",
+    "@types/sinon-chai": "^3.2.5",
+    "@types/webpack": "^4.41.22",
     "chai": "^4.1.2",
-    "codecov-lite": "^0.1.3",
-    "codelyzer": "^4.1.0",
-    "commitizen": "^2.8.1",
-    "concurrently": "^3.0.0",
-    "copyfiles": "^1.2.0",
-    "core-js": "^2.5.3",
-    "cz-conventional-changelog": "^2.0.0",
-    "del-cli": "^1.1.0",
-    "fork-ts-checker-webpack-plugin": "^0.3.0",
-    "html-webpack-plugin": "^2.28.0",
-    "husky": "^0.14.3",
+    "codecov-lite": "^2.0.0",
+    "codelyzer": "^6.0.1",
+    "commitizen": "^4.2.1",
+    "concurrently": "^5.3.0",
+    "copyfiles": "^2.4.0",
+    "core-js": "^3.6.5",
+    "cz-conventional-changelog": "^3.3.0",
+    "del-cli": "^3.0.1",
+    "fork-ts-checker-webpack-plugin": "^3.1.1",
+    "html-webpack-plugin": "^4.5.0",
+    "husky": "^4.3.0",
     "istanbul-instrumenter-loader": "^3.0.0",
-    "karma": "^2.0.0",
-    "karma-chrome-launcher": "^2.2.0",
-    "karma-coverage-istanbul-reporter": "^1.3.3",
-    "karma-mocha": "^1.3.0",
+    "karma": "^5.2.3",
+    "karma-chrome-launcher": "^3.1.0",
+    "karma-coverage-istanbul-reporter": "^3.0.3",
+    "karma-mocha": "^2.0.1",
     "karma-sourcemap-loader": "^0.3.7",
-    "karma-webpack": "^2.0.9",
-    "lint-staged": "^6.0.1",
-    "mocha": "^5.0.0",
-    "ng-packagr": "2.0.0-rc.11",
-    "prettier": "^1.10.2",
-    "rxjs": "^5.5.6",
-    "sinon": "^4.1.6",
-    "sinon-chai": "^2.8.0",
-    "standard-version": "^4.3.0",
-    "ts-loader": "^3.2.0",
-    "ts-node": "^4.1.0",
-    "tslint": "^5.9.1",
-    "tslint-config-mwl": "^0.3.0",
+    "karma-webpack": "^4.0.2",
+    "lint-staged": "^10.4.0",
+    "mocha": "^8.1.3",
+    "ng-packagr": "^10.1.2",
+    "prettier": "^2.1.2",
+    "rxjs": "^6.6.3",
+    "rxjs-compat": "^6.0.0-rc.0",
+    "sinon": "^9.2.0",
+    "sinon-chai": "^3.5.0",
+    "standard-version": "^9.0.0",
+    "svg-gauge": "^1.0.6",
+    "ts-loader": "^8.0.4",
+    "ts-node": "^9.0.0",
+    "tslint": "^6.1.3",
+    "tslint-config-mwl": "^0.8.1",
     "tslint-loader": "^3.5.2",
-    "typescript": "~2.6.2",
+    "typescript": "~4.0.3",
     "validate-commit-msg": "^2.12.1",
-    "webpack": "^3.10.0",
+    "webpack": "^4.44.2",
     "webpack-angular-externals": "^1.0.0",
-    "webpack-dev-server": "^2.11.0",
-    "webpack-rxjs-externals": "^1.0.0",
-    "zone.js": "^0.8.20"
+    "webpack-cli": "^3.3.12",
+    "webpack-dev-server": "^3.11.0",
+    "webpack-rxjs-externals": "^2.0.0",
+    "zone.js": "^0.11.1"
   },
   "peerDependencies": {
-    "@angular/core": ">=5.0.0"
+    "@angular/core": ">=10.0.0",
+    "svg-gauge": "^1.0.6"
```

Comment:
Can we keep `svg-gauge` out of peer dependencies? Just one less thing for people to worry about in their package.json as we already install it for them as a dependency.

Reply:
Good call. Updated.